app/javascript/shared/components/focusTrap.jsx

```diff
@@ -1,27 +1,62 @@
 import PropTypes from 'prop-types';
+import { h, Fragment } from 'preact';
 import { useLayoutEffect, useRef } from 'preact/hooks';
 import { createFocusTrap } from 'focus-trap';
 import { defaultChildrenPropTypes } from '../../common-prop-types';
+import { KeyboardShortcuts } from './useKeyboardShortcuts';
 
-export const FocusTrap = ({ selector, children }) => {
+export const FocusTrap = ({ selector, children, onDeactivate }) => {
```

Comment:
documentation (blocking): Exported functions require JS Doc documentation. For an example, see https://github.com/forem/forem/blob/master/app/javascript/utilities/http/request.js#L1-L23